src/components/app/app.scss

```diff
@@ -8,18 +8,7 @@
     flex-direction: column;
 
     &-content {
-      display: flex;
-      flex-direction: column;
       height: 100%;
-      padding: 15px 40px 15px 40px;
-      background-color: $color-primary;
-      width: 100%;
     }
```

Comment:
maybe I should leave it here:
padding: 15px 40px 15px 40px;
background-color: $color-primary; ?

and we don't need width, because of display:block by default of app__container-content?